### ISSUE TITLE
Can't reset password

### ISSUE THREAD

**Issue by nhoknakroz** [phpcollab/phpcollab#82]
Hi @mindblender ,

I've run a fresh installation of phpCollab version 2.10.3.
When I'm trying to reset password of an user, it always return the message "You have tried too many times. Please wait and try again".
I've made a simple fix so the feature worked but I think you should consider about this problem and make an official fix.

Change in `classes/Members/ResetPassword.php` at line 66:
```php
if ($this->token && $this->isTimestampExpired($this->timestamp, $times['tokenLifespan'])) {
if (!$this->token || $this->isTimestampExpired($this->timestamp, $times['tokenLifespan'])) {  // FIX
```
Change in `classes/Members/ResetPassword.php` at line 125:
```php
if ($this->isTimestampExpired($this->timestamp)) {
if (!$this->isTimestampExpired($this->timestamp)) {  // FIX
```
Delete line 226, 227 in `classes/Members/ResetPassword.php`.

Change in `general/resetpassword.php` at line 11:
```php
$token = $request->query->get("token");
$token = $request->query->get("token") ?? $request->request->get("token");  // FIX
```
Change in `general/resetpassword.php` at line 103:
```php
$block1->openForm("../general/sendpassword.php", null, $csrfHandler);
$block1->openForm("../general/resetpassword.php", null, $csrfHandler);  // FIX
```

**Comment by mindblender** [comment]
Hi @nhoknakroz,

Thanks for the information. I would appreciate it if you could please
submit a PR with your code changes.

-Jeff

**Comment by nhoknakroz** [comment]
Hi @mindblender, are there any updates? Can I close this issue?

**Comment by mindblender** [comment]
Let me review it, your changes and mine, this week and I'll close it this weekend if the problem is fixed.  Thanks!

**Comment by nhoknakroz** [comment]
Hi @mindblender. Do you have any updates?

**Comment by mindblender** [comment]
Sorry, Haven't had time to work on this in a few weeks.  I made the suggested changes and merged PR #84.  Thanks!
